MobiFlight/Board.cs

```diff
@@ -91,6 +101,11 @@ public class Connection
         /// Maximum size of a CmdMessenger message, in bytes.
         /// </summary>
         public int MessageSize;
+
+        /// <summary>
+        /// Number of milliseconds to wait for the firmware update to complete before attempting to call GetInfo on the board.
+        /// </summary>
+        public int TimeoutForFirmwareUpdate = 15000;
```

Comment:
`15000` is the original hardcoded value so it's used as the default here. If it is omitted from the `.board.json` file the default gets used.